### DISCUSSION TITLE
deploying-asp-net-core-applications-to-kubernetes-part-9-monitoring-helm-releases-that-use-jobs-and-init-containers/

### DISCUSSION THREAD

**Discussion by andrewlock** [andrewlock/blog-comments#342] [Announcements]
# deploying-asp-net-core-applications-to-kubernetes-part-9-monitoring-helm-releases-that-use-jobs-and-init-containers/

In this post I show how to detect when a release that uses jobs and init containers for running database migrations has successfully deployed

https://andrewlock.net/deploying-asp-net-core-applications-to-kubernetes-part-9-monitoring-helm-releases-that-use-jobs-and-init-containers/

[a comment by an automated account is not shown]

**Comment by akhanalcs** [comment]
Is this `.sh` script still the recommended approach in 2024?
